src/redux/actions.ts

```diff
@@ -1,9 +1,6 @@
 import { AppAction, CHANGE_SECTION, SectionName } from './actionTypes';
 
-function changeSection(sectionName: SectionName): AppAction {
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
oh is this complaining that we should do `export default`? we could keep how it originally was:
```
export default {		
   changeSection,		
 };
```

Reply:
nah it makes sense to export like this, it's only complaining because we only have 1 function. If we add more, it'll stop complaining. Let's just keep the disable until we add another function?

Reply:
ah i see. i think you could also make it a closure and do `export const changeSection = (sectionName: SectionName): AppAction => { }`. i actually think closures are preferred